### DISCUSSION TITLE
Is it possible to stream the captured audio to Home Assistant?

### DISCUSSION THREAD

**Discussion by regystro** [stas-sl/esphome-sound-level-meter#17] [Q&A]
Hi.

I know this is not the purpose of this cool project, but could anybody think of a way to capture the audio and stream it to Home Assistant, so we can feed it to any media_player device?

[Voice Assistant](https://esphome.io/components/voice_assistant.html) already does it, but I cannot find a way to use the audio for another purposes apart from STT or even replay it in HA. Thus I'm looking for another approaches, like streaming to FFmpeg integration or something like that.

Any ideas are welcome. Thanks!!

**Comment by RaymiiOrg** [comment]
Here is a topic on the home assistant forums that uses the "official" i2s components (not these) to stream the data to a UDP receiver, for example, VLC:

https://community.home-assistant.io/t/is-there-a-way-to-stream-audio-from-one-esphome-to-another/581786
